sw-emulator/lib/periph/src/ml_dsa87.rs

```diff
@@ -557,15 +562,20 @@ mod tests {
 
         let mut ml_dsa87 = Mldsa87::new(&clock, key_vault);
 
-        let mut seed = [0u8; 32];
+        let mut seed = rand::thread_rng().gen::<[u8; 32]>();
         seed.to_big_endian(); // Change DWORDs to big-endian. TODO is this needed?
```

Comment:
This step can be removed as it does not matter for a test generated seed. A seed coming from the KV needs to have it's DWORD endianness reversed when passing to the FIPS204 keygen function.